builds/azure-pipelines/build-release.yml

```diff
@@ -54,9 +51,13 @@ stages:
           # Disabling integration tests on macOS due to issues with running MSSQL on Docker
           # We need to set up a self-hosted agent with Docker running by default: https://github.com/microsoft/azure-pipelines-tasks/issues/12823
           testServer: ''
+          Codeql.BuildIdentifier: extension_mac
+          Codeql.Language: csharp,java,javascript,powershell,python,tsql
```

Comment:
Note from CodeQL docs: "If repository contains an interpreted language, set Codeql.Language variable for that language in one of the stages/jobs to get full coverage."

I am setting the language for javascript, powershell, python, tsql only in the Mac job.